peps/pep-0752.rst

```diff
@@ -0,0 +1,399 @@
+
+* The organization manually removes themselves as the owner.
+* The organization is deleted.
+
+When a reserved namespace becomes unclaimed, repositories:
+
+1. MUST remove the `visual indicator <user-interface_>`_
+2. MUST NOT modify past `release metadata <repository-metadata_>`_
+
+Grant Approval Criteria
+-----------------------
+
+Applications lacking substantial evidence of satisfying the following criteria
+are likely to be rejected:
+
+1. Does the namespace properly and clearly identify the reservation owner?
+2. Is the namespace something common that should not belong to any individual
+   owner or organization? Avoid namespaces that are shorter than four
+   characters and avoid common or generic words.
+3. Would *not* reserving the namespace cause ambiguity, confusion, or other
+   harm to the community?
+4. Is the organization actively using the namespace?
+
+By default, `root grant <root-grant_>`_ are only available for
+`corporate organizations <corp-orgs_>`_. Other organizations may be considered
+on a case-by-case basis, some examples:
+
+* Large, popular open-source projects with many packages like
+  `OpenTelemetry`__
+* Universities that actively publish packages
+* NPOs/NGOs that actively publish packages like
+  `Our World in Data <https://github.com/owid>`__
+* Government organizations that actively publish packages
+
+__ https://github.com/open-telemetry/opentelemetry-python-contrib/tree/main/instrumentation
+
+Backwards Compatibility
+=======================
+
+There are no intrinsic concerns because there is still a flat namespace and
+installers need no modification. Additionally, many projects have already
+chosen to signal a shared purpose with a prefix like `typeshed has done`__.
+
+__ https://github.com/python/typeshed/issues/2491#issuecomment-578456045
+
+Security Implications
+=====================
+
+* Although users will no longer see the visual indicator when a namespace
+  becomes unclaimed, external consumers of metadata may have difficulty
+  scraping the user facing
+  `enumeration <user-interface_>`_ of grants to verify current ownership.
+* There is an opportunity to build on top of :pep:`740` and :pep:`480` so that
+  one could prove cryptographically that a specific release came from an owner
+  of the associated namespace. This PEP makes no effort to describe how this
+  will happen other than that work is planned for the future.
+
+How to Teach This
+=================
+
+For organizations, we will document how to reserve namespaces, what the
+benefits are and pricing.
+
+For consumers of packages we will document the indicator on release pages, how
+metadata is exposed in the `API <repository-metadata_>`_ and potentially in
+future note tooling that supports utilizing namespaces to provide extra
+security guarantees during installation.
+
+Reference Implementation
+========================
+
+None at this time.
+
+Rejected Ideas
+==============
+
+Allow Private Namespaces for Community Projects
```

Comment:
I need to re-read, but is this the first mention of "Private Namespaces"?

Reply:
I changed it back to what I had originally before I opened the PR which was `Non-Public` 🙂